### PR TITLE
DEVOPS-47: Creating custom groovy slack notification function

### PR DESCRIPTION
**Pull request by lenardw1**

### CODE REVIEW

File: vars/sendToSlack.groovy [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    if (messageType == 'success') {
+        color = '#008000'
+    } else if (messageType == 'inProgress') {
+        color = '#FFFF00'
+    }
+    else if (messageType == 'failure') {
+        color = '#FF0000'
+    }
+    
+    slackSend (color: ${color}, message: ${message}: `${env.JOB_NAME}` [Build #${env.BUILD_NUMBER}] \n(<${env.BUILD_URL}|Open>)")
```
Comment (RussianPrussian) [inline vars/sendToSlack.groovy:14]: do you know that you'll have access to the environment variables in the function?

Comment (lenardw1) [reply vars/sendToSlack.groovy:14]: Yes in a def call you have access to them. You dont in a class however.

Comment (RussianPrussian) [reply vars/sendToSlack.groovy:14]: Okay.

Comment (RussianPrussian) [reply vars/sendToSlack.groovy:14]: Test it brother. LGTM

**Review by RussianPrussian — APPROVED** [review]
LGTM

**Pull request merged**